pyproject.toml

```diff
@@ -1,13 +1,14 @@
 [build-system]
-requires = ["poetry-core"]
-build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core>=1.0.0", "poetry-dynamic-versioning>=1.0.0,<2.0.0"]
+build-backend = "poetry_dynamic_versioning.backend"
 
 [tool.poetry]
 name = "imap-processing"
-version = "0.2.0"
+version = "0.0.0"
```

Comment:
This gets changed during build/release, but not committed to the repository.

Reply:
Add a comment that this is updated during release?

Reply:
Added!